packages/components/src/components/Calculator/index.tsx

```diff
@@ -212,19 +222,25 @@ export const Calculator: FC<Props> = ({
   const { calculator, form, inputResults, processAllFieldCodes } =
     useCalculator(valueWithContext, _environment);
 
+  const _settings: PlaygroundSettings = useDeepCompareMemoize(settings);
```

Comment:
I don't think `useDeepCompareMemoize` is necessary anymore.

Reply:
I did some investigation yesterday, and this is caused by `getAdjustedMergedSettings` and `getMergedSettings` which create a new settings object in each render.

I've started to refactor `ValueWithContextViewer` to fix this, will finish today.